app/routes/profile/index.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model(params) {
+    return Ember.RSVP.hash({
```

Comment:
i think you'd do `this.modelFor('profile')`. the implementation of `profile.index` is the same as `profile`. 

thinking about it more, as its done now this is basically a no-op because the promise for the parent route will resolve first and these records will be primed in the store. do we even need the parent route?